_extras/Recipe.md

```diff
@@ -0,0 +1,76 @@
+* **Customize profile names** to clearly indicate instructors and helpers. For example, a helper named Ada might change their profile to read "Helper: Ada (she/her)"
+* **Use Zoom features as you would use sticky notes**, and remember to call upon learners to use them regularly. 
+    * On Zoom, learners can choose "reactions" that include a green '✓' and a red 'x' icon. Unlike other reactions, these persist until cleared.
+* **Use breakout rooms** for socialization and activities whenever possible. These do take time and that may mean cutting content. The benefits to creating a supportive learning environment make this cost worthwhile.
+    * Using a shared timer can help, especially as learners get used to using it.
+* **Ask learners to keep the chat focused.** You may wish to provide them with an alternate venue to share social chat.
+* **Use a 3-step help response system**.
+    * Step 1: Learners message a helper, who helps via **direct messaging** in the conference chat if they can. 
+    * Step 2: Helpers communicate about questions they have received via **back-channel conversations** (even if resolved). This will elevate common questions that merit attention of the teaching Instructor and will allow others to advise if the problem has not been solved.
+    * Step 3: For difficult problems, invite the learner to **share their screen** for live support. Depending on the workshop size, frequency of the problem, etc., this may be done in the main room as a pause in the workshop or in a breakout room with a helper.
+        * If you create breakout rooms for activities, it helps to create extra rooms for this purpose at that time so that activity rooms can be preserved.
+        * Keep in mind that learners in breakout rooms will miss ongoing instruction in the main room, so helpers should also help learners to catch up before returning.
+* **Divert distractions and disruptions to breaks** wherever possible. Advanced questions and other difficult conversations can be resolved using breakout rooms during breaks. If you have plenty of breaks available, missing one to problem solve will be less strenuous.
+
+## 3. Assessment
+* **Formative assessment is more important than ever**. When teaching online, you lose the ability to visually "read the room," especially if learners' cameras are turned off. That's ok, because frequent formative assessment will provide the feedback you need. 
+    * Ask a question that learners can answer in the chat.
+    * Prepare a polling tool or Google form with a few questions in advance.
+    * Use virtual sticky notes to check pacing - 'green check' for 'fine/too slow' and 'red x' for too fast.
+    * Use 'virtual minute cards' for +/- feedback or to ask learners to share 'one thing you have learned'/'one question you still have' at break points
```

Comment:
I believe there is a virtual minute card template. Maybe  we can link that here

Reply:
Using the nonverbal prompts in Zoom is also helpful for checking in on learners

Reply:
The green check and red x are permanent, so I usually have to remind people to click again after an exercise is finished or the problem is solved.

Reply:
Link to virtual minute card template - https://docs.google.com/forms/d/1p7iOV5HNvy4POS4g6eottY8RSfKq4kaoKz1-jIFYTMI/template/preview